draft-ietf-teep-protocol.md

```diff
@@ -1186,3 +1186,179 @@ ta-uuid = 16
 ta-manifest-sequence-number = 17
 have-binary = 18
 ~~~~
+
+# D. Examples of Diagnostic Notation and Binary Representation {#examples}
+{: numbered='no'}
+
+### Some assumptions in examples
+
+- TAM supports 2 versions: 0, 1
+- OCSP stapling data = h'010203'
+- TEEP Device has 2 TAs
+  - TA-ID: 0x010203, 0x040506
+- SUIT manifest-list is set empty only for example purpose
```

Comment:
I'm all for having examples, but in my view examples should be realistic, not show something that would never appear on the wire.

Reply:
I have to ask somebody to add the SUIT manifest-list example later :)

Reply:
```suggestion
- SUIT manifest-list is set empty only for example purposes
```